common/src/View/DuplicateNodesCommand.cpp

```diff
@@ -53,15 +55,17 @@ namespace TrenchBroom {
 
                     Model::Node* parent = original->parent();
                     if (cloneParent(parent)) {
-                        auto insertPos = MapUtils::findInsertPos(newParentMap, parent);
```

Comment:
I still don't understand what is going on here, why is it using lower_bound and what does std::prev(hint) represent?

Reply:
The code looks like this because I wanted to avoid doing two lookups in the map, one for checking if the parent is already cloned and then again on insertion if it turned out that it hasn't been cloned yet. But you are right in that it makes the code more complicated at little gain. I'll change it so that its easier to understand.

Reply:
done